editors/vscode/src/common.ts

```diff
@@ -130,6 +123,18 @@ export function languageClientOptions(
     };
 }
 
+export function use_wasm_preview(): boolean {
```

Comment:
This function only makes sense for the native extension.
When using the browser extension, it should always use the wasm_preview,  what mechanisms is forcing it now?